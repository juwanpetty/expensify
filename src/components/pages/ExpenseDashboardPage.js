import React from "react";
import ExpenseList from "../expense/ExpenseList";
import ExpenseListFilters from "../expense/ExpenseListFilters";

const ExpenseDashboardPage = () => {
  return (
    <div>
      <ExpenseListFilters />
      <ExpenseList />
    </div>
  );
};

export default ExpenseDashboardPage;
