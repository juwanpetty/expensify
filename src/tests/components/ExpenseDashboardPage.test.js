import React from "react";
import { shallow } from "enzyme";
import ExpenseDashboardPage from "../../components/pages/ExpenseDashboardPage";

test("should render expense dashboard page item correctly", () => {
  const wrapper = shallow(<ExpenseDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
