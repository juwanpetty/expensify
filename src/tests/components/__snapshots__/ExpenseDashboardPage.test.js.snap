// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense dashboard page item correctly 1`] = `ShallowWrapper {}`;
